examples/EMR_Cluster.py

```diff
@@ -90,7 +92,7 @@
     BootstrapActions=[emr.BootstrapActionConfig(
         Name='Dummy bootstrap action',
         ScriptBootstrapAction=emr.ScriptBootstrapActionConfig(
-            Path='/bin/sh',
+            Path='file:/bin/sh',
```

Comment:
How do you know the `file:` change is required? It makes sense but I don't see it in the AWS documentation on http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-emr-cluster-bootstrapactionconfig-scriptbootstrapactionconfig.html. When you stood up this template, did you verify that this worked?